worker.go

```diff
@@ -140,6 +172,9 @@ func (w *Worker) Flush() WorkerMetrics {
 		1.0,
 	)
 
+	w.stats.Count("worker.metrics_processed_total", processed, []string{fmt.Sprintf("worker:%d", w.id)}, 1.0)
```

Comment:
Per your point above about the flush not using the values, I agree. But it would simplify this method if you didn't have to zero the `w.<counter>` and then could use `ret.processed` here.

I defer to your opinion though, this is a nit.